source/index.md

```diff
@@ -926,6 +926,9 @@ Declined   |  40000 |  General input error
            |  40413 |  Insufficient funds
            |  40414 |  Suspected fraud
            |  40415 |  Amount limit exceeded
+           |  40420 |  R0 Stop request. Stop this subscription on this merchant
```

Comment:
Will it confuse if we start referring to scheme response codes (i.e. `R0-R2`)? Do we to mention e.g. `R0`?

Reply:
Looking at the existing entries in this table, I think we want the text in the Meaning column to match what the gateway returns, i.e. what we have in `.../transaction/status.rb` in the database layer. The Rx response codes are not mentioned in the messages returned by the gateway—at least in the current version of the database layer pull request.

Reply:
Direct scheme response codes has been removed. R0, R1, R3 are Visa specific, so this solution would leave Mastercard unresolved. Instead are more general concept of **stop** should be adapted.